testsuite/MDAnalysisTests/coordinates/test_pdb.py

```diff
@@ -961,3 +961,10 @@ def test_partially_missing_cryst():
     assert len(u.atoms) == 3
     assert len(u.trajectory) == 2
     assert_array_almost_equal(u.dimensions, 0.0)
+
+def test_CRYST1_Warning():
+    #issue 2599  
+    with pytest.warns(UserWarning) as record:
+        u = mda.Universe(CRYST1_BOX)
+        cur_sele = u.select_atoms('around 0.1 (resid 4 and name CA and segid A)') 
```

Comment:
You _shouldn't_ need to call `select_atoms` to get the warning here (it should hopefully just be thrown on universe building (worth checking that's the case here). So the `select_atoms` shouldn't be necessary here. However, we definitely should be checking that selecting 0.1 A around one of the alpha carbons works now, so do please add this as a second test.